examples/catalog/src/top-bar/TopBar.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import MaterialIcon from '@material/react-material-icon';
+import TopAppBar, {
+  TopAppBarIcon,
+  TopAppBarRow,
+  TopAppBarSection,
+  TopAppBarTitle,
+} from '@material/react-top-app-bar';
+
+import {MenuContext} from '../Context';
+import {Repository} from '../Router';
+
+const styles = require('./TopBar.scss');
+
+export const TopBar = () => (
+  <MenuContext.Consumer>
+    {({setOpen}) => (
+      <TopAppBar>
+        <TopAppBarRow>
+          <TopAppBarSection align='start'>
+            <TopAppBarIcon navIcon tabIndex={0}>
+              <MaterialIcon
+                hasRipple
+                icon='menu'
+                onClick={() => setOpen(true)}
```

Comment:
move this out to its own function as well.

Reply:
This is also same issue with above, right?

Reply:
correct

Reply:
I fixed it!